Tests/OpenAPIGeneratorReferenceTests/SnippetBasedReferenceTests.swift

```diff
@@ -2514,6 +2514,18 @@ final class SnippetBasedReferenceTests: XCTestCase {
                       type: array
                       items:
                         type: string
+                  - name: sort
+                    in: query
+                    required: true
```

Comment:
```suggestion
                    required: false
```

Let's use `required: false`, or if you'd like to test it being required, make one of the properties required.